filament/backend/src/vulkan/VulkanContext.cpp

```diff
@@ -546,7 +548,7 @@ void flushWorkCommandBuffer(VulkanContext& context) {
     };
     vkEndCommandBuffer(work.cmdbuffer);
     vkQueueSubmit(context.graphicsQueue, 1, &submitInfo, work.fence->fence);
-    work.submitted = true;
+    work.fence->submitted = true;
```

Comment:
Do you need to lock / unlock the mutex here?

Reply:
This is the "work" command buffer which the DriverAPI client should never wait on.

Reply:
Got it!